CONTRIBUTING.md

```diff
@@ -0,0 +1,130 @@
+also encourage you to comment on any existing issues. Each issue forms a special thread for clearly delimited
+discussion.
+
+### How to raise an issue within an interest group?
+
+Post your issue to the **issue tracker** pertaining to your interest group, or to the
+[central](https://github.com/CLARIAH/IG/issues) group if it is more generic and spans multiple interest groups, or to
+the [use cases](https://github.com/CLARIAH/usecases/issues) issues if it specifically deals with use case.
+
+### How to add a use case?
+
+See [https://github.com/CLARIAH/usecases/blob/master/CONTRIBUTING.md](here) .
+
+### How to raise an issue about a use case?
+
+Open an issue on the [issue tracker for use cases](https://github.com/CLARIAH/usecases/issues)
+
+### Can we organize our documents in a DropBox/GoogleDrive/SURFdrive?
+
+No, please don't do that and keep them in the proper git repository instead to facilitate transparency. Otherwise we
+would have documents or copies of documents all over the place and we don't know where to look to find them or whether
+they are the proper latest version. Git is purposefully designed to solve this exact issue. The whole idea of using git
+and github for the interest groups is to consolidate version control and have a single public entry point.
+
+If you do have the document elsewhere, at least put a link to the document in the git repository (see next point).
+
+### We have a document on Google Docs/Office 365, can we link to it?
+
+Yes, you can link to the document from the git repository by adding simple markdown document to the git repository
+that contains a link to the document on the external platform. (be aware not to share a link with public write
+permissions in that case).
+
+Preferably, you can export the document and put it in the git repository itself. But you will have to repeat the
+procedure every time the document on the external platform changes. The version in git should always be authoritative.
+
+### Can't we just mail documents around to participants?
+
+No, this does not facilitate transparency, findability, reusability. Please use the repository and mail links to the
+document in the repository if needed.
+
+The notable exception here is when it concerns privacy-sensitive material, in that case the git repositories are **NOT**
+suitable.
+
+### Does everything have to be in Markdown format?
+
+No, we recommend to do as much as possible in Markdown because it is a very simple and accessible text-based format with
+almost no learning curve. It can be very easily visualised and easily converted to richer formats (like LaTeX, PDF). If
+Markdown is not sufficient for your needs because you deal for example with complex spreadsheets, matrices and/or
+large tables, or you're writing a journal/conference paper, then feel free to use other formats.
+
+Our recommendation in this latter case is to use LaTeX, the tex sources and all required assets can be maintained in the git
```

Comment:
For spreadsheets, I'd recommend _against_ latex (although there is probably a package for it :stuck_out_tongue: )

Reply:
Yes 😄 "the latter" refers to papers of course, definitely not to spreadsheets